lib/validators/movie.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
```

Comment:
did the guide tell you to add test for this validator?

Reply:
It's the next step, but i'm gonna add it to this PR to group validation together

Reply:
added the validation tests